radixdlt-core/radixdlt/src/main/java/com/radixdlt/sync/RemoteSyncService.java

```diff
@@ -133,6 +155,31 @@ private void processLedgerUpdate(LedgerUpdate ledgerUpdate) {
 		final VerifiedLedgerHeaderAndProof updatedHeader = ledgerUpdate.getTail();
 		if (accComparator.compare(updatedHeader.getAccumulatorState(), this.currentHeader.getAccumulatorState()) > 0) {
 			this.currentHeader = updatedHeader;
+			ledgerUpdate.getNextValidatorSet().ifPresent(newValidatorSet -> this.currentValidatorSet = newValidatorSet);
+			this.sendStatusUpdateToSomePeers(updatedHeader);
+		}
+	}
+
+	private void sendStatusUpdateToSomePeers(VerifiedLedgerHeaderAndProof header) {
+		if (!(this.localSyncService.getSyncState() instanceof SyncState.IdleState)) {
+			return; // not sending any updates if the node is syncing itself
 		}
+
+		final var statusUpdate = LedgerStatusUpdate.create(header);
+
+		final var nonValidatorPeers = this.peersView.peers().stream()
```

Comment:
Not sure if we need to only send to non-validating peers. It is helpful to have the sync layer just be an overall sync layer.

Reply:
validators don't need that, they can just sync up based on received votes/proposals